tasks/restore.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Ensure directory for snapshot restore exists
+  file:
+    path: "{{ backups_role_restore_path }}"
+    state: directory
+    owner: "{{ backups_role_user_name }}"
+    group: "{{ backups_role_user_group }}"
+    mode: 0775
+
+- name: Restore last backup for this restic repo
+  command: "restic-{{ backups_role_restic_repo_name }} restore latest --target {{ backups_role_restore_path }}"
```

Comment:
I don't understand the command that would be generated

Reply:
Here I'm not using `restic` directly, but the restic helper that ansible-restic is creating. It is a bash script that keeps the credentials and calls restic with whichever parameters you call it. [restic-helper template](https://github.com/paulfantom/ansible-restic/blob/master/templates/restic.helper.j2)

The name is chosen in a [task](https://github.com/paulfantom/ansible-restic/blob/master/tasks/configure.yml#L11), using the template module. It is `restic-<RESTIC-REPO-NAME>`. It has a variable name because ansible-restic allows you to configure multiple restic-repos in the same machine, hence the loop `with_items: '{{ restic_repos }}'`. In our case, though (backups_role), we only use one repo, so [I chose to use the hostname as the restic repo name](https://github.com/coopdevs/backups_role/blob/master/defaults/main.yml#L12).

Reply:
OK, would you mind to add a link or something at line 9 to document a bit this? Thanks!